packages/react-composites/src/composites/CallComposite/pages/ConfigurationPage.tsx

```diff
@@ -206,7 +218,25 @@ export const ConfigurationPage = (props: ConfigurationPageProps): JSX.Element =>
   const forceShowingCheckPermissions = !minimumFallbackTimerElapsed;
 
   /* @conditional-compile-remove(video-background-effects) */
-  const { toggleVideoEffectsPane } = useVideoEffectsPane(props.updateSidePaneRenderer, mobileView);
+  const [dismissedVideoEffectsError, setDismissedVideoEffectsError] = useState<DismissedError>();
+  /* @conditional-compile-remove(video-background-effects) */
+  const onDismissVideoEffectError = useCallback((error: AdapterError) => {
+    setDismissedVideoEffectsError(dismissVideoEffectsError(error));
+  }, []);
+  /* @conditional-compile-remove(video-background-effects) */
+  const latestVideoEffectError = useSelector(videoBackgroundErrorsSelector);
+  /* @conditional-compile-remove(video-background-effects) */
+  const showVideoEffectError =
```

Comment:
nit: instead of passing `showError : true | false` to the videoeffectspane and having the video effects pane have to use a selector to retreive the error, the errors are already retrieved here so the logic would be streamlined if this passed the error to videoPane instead of a showError boolean